webapp/src/components/surveyPost/textQuestion/style.scss

```diff
@@ -0,0 +1,8 @@
+.TextQuestion {
+    .questionTitle {
+        color: #3D3C40;
```

Comment:
Is this going to be visible on different themes?

Reply:
It don't. I have a separate ticket to fix the colors here and in system console. I'll create a separate PR to fix the colors.